docs/faq.rst

```diff
@@ -1,11 +1,32 @@
 FAQs
 ====
 
-How does Mantl compare to `OpenStack Magnum <https://wiki.openstack.org/wiki/Magnum>`_?
+How are Mantl components chosen_?
+--------------------------------------------------------------------------------------
+
+A service design methodology was used to assemble Mantl components. Priority was given
+to sensing the large shifts in this technology and to defining an architecture 
+consistent with mode 1 to mode 2+ methodologies. Once the architecture was optimized 
```

Comment:
We shouldn't expect that anyone will know that terminology, I've personally never heard it.